geode-junit/src/main/java/org/apache/geode/test/version/VersionManager.java

```diff
@@ -180,12 +195,15 @@ private void findInstalls(String fileName) {
 
   private Optional<String> parseVersion(String version) {
     String parsedVersion = null;
-    if (version.startsWith("test") && version.length() >= "test".length()) {
-      if (version.equals("test")) {
-        parsedVersion = CURRENT_VERSION;
-      } else {
-        parsedVersion = version.substring("test".length());
-      }
+    if (version.length() > 0 && Character.isDigit(version.charAt(0))
+        && version.length() >= "1.2.3".length()) {
+      // for (int i = 1; i < version.length(); i++) {
```

Comment:
Some new commented out code here we probably want to remove before merging (assuming its not needed anymore)

Reply:
okay